src/main/java/no/unit/nva/bare/CreateAuthorityHandler.java

```diff
@@ -66,28 +70,28 @@ protected GatewayResponse createAuthorityOnBare(String name) {
         try {
             HttpResponse<String> response = bareConnection.createAuthority(bareAuthority);
```

Comment:
Denne må splittes opp, altfor mye som skjer i en metode og altfor mye nøsting.

Reply:
I have created a task for upgrading this handler (outside scope of task at hand, but changes in dependencies forced a few changes nonetheless)

https://unit.atlassian.net/browse/NP-1063